core/server/api_container/server/startosis_engine/startosis_warning/warning.go

```diff
@@ -0,0 +1,38 @@
+package startosis_warning
```

Comment:
uncomfortable with this PR as this isn't tested. I could have lived with that if I had seen a working demo!

Reply:
I demoed it today - that should be it? @h4ck3rk3y

Reply:
sure. can we pop in a unit test if its like <10 minutes of work? otherwise i have approved this!

Reply:
absolutely, should have a test